spark/src/main/java/org/apache/spark/CometTaskMemoryManager.java

```diff
@@ -40,7 +40,7 @@ public class CometTaskMemoryManager {
   /** The id uniquely identifies the native plan this memory manager is associated to */
   private final long id;
 
-  private final TaskMemoryManager internal;
+  public final TaskMemoryManager internal;
```

Comment:
we prob should have another name for that?

Reply:
yes, probably